apps/webcert/src/App.tsx

```diff
@@ -10,23 +10,27 @@ import SubscriptionWarningModal from './feature/subscription/SubscriptionWarning
 import { throwError } from './store/error/errorActions'
 import { createErrorRequest } from './store/error/errorCreator'
 import { ErrorCode, ErrorType } from './store/error/errorReducer'
-import { useAppDispatch } from './store/store'
+import { useAppDispatch, useAppSelector } from './store/store'
 import { cancelLogout, triggerLogout } from './store/user/userActions'
 import { initateApplication } from './store/welcome/welcomeActions'
+import { getUser } from './store/user/userSelectors'
 
 function App(): JSX.Element {
   const dispatch = useAppDispatch()
+  const origin = useAppSelector(getUser)?.origin
```

Comment:
This could cause unnecessary renders, please change to `useAppSelector((state) => getUser(state).origin)`

Reply:
Thanks, I will make the change